src/execution/operator/helper/physical_limit.cpp

```diff
@@ -2,6 +2,8 @@
 
 #include "duckdb/common/algorithm.hpp"
 
+#include <duckdb/execution/expression_executor.hpp>
```

Comment:
Use `""` for internal libraries, and `<>` for external libraries. This is important because the amalgamation script relies on this distinction as part of the.... professionally written regex.

Reply:
Yeah I know, sometimes the IDE automatically includes stuff,